global.d.ts

```diff
@@ -15,7 +15,7 @@ declare global {
       apiVersion?: '1';
       clientSideEncryption?: boolean;
       serverless?: 'forbid' | 'allow' | 'require';
-      auth: 'enabled' | 'disabled';
+      auth?: 'enabled' | 'disabled';
```

Comment:
Also included in some of Neal's PRs.  If those PRs merge first, I'll rebase main to pull it in.

This fixes type errors caused by our custom overloads to Mocha's `it`.